consent/strategy_default.go

```diff
@@ -983,6 +983,37 @@ func (s *DefaultStrategy) HandleOpenIDConnectLogout(ctx context.Context, w http.
 	return s.completeLogout(ctx, w, r)
 }
 
+func (s *DefaultStrategy) HandleHeadlessLogout(ctx context.Context, w http.ResponseWriter, r *http.Request, sid string) error {
+	loginSession, lsErr := s.r.ConsentManager().GetRememberedLoginSession(ctx, sid)
+
+	// This is ok (session probably already revoked), do nothing!
+	if lsErr != nil {
+		return nil
+	}
+
+	if err := s.executeBackChannelLogout(r.Context(), r, loginSession.Subject, sid); err != nil {
```

Comment:
Everything from here until the end of the function is the same as in `completeLogout` function. Is this the correct direction to take here?

Reply:
That looks good to me! Since this is called via an API, we won't have front-channel log-out (because no browser is involved). I think this is fine, but should probably be clarified in the API / function doc. You may opt to create a new function with the flow as to avoid duplication, but it's not required since it's only 2 functional statements